src/index.rs

```diff
@@ -253,6 +263,10 @@ impl VariableIndexEntry {
         self.linkage == LinkageType::External
     }
 
+    pub fn is_var_external(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_external(&self) -> bool {
```

Reply:
and maybe rename the previous `is_external` function to `is_external_linkage` then?

Reply:
Honestly I'd prefer to keep the external linkage function as `is_external` since, at least in my mind, if something says it is external I'd assume linkage. This might make the code harder to follow when encountering `is_external` somewhere and it refers to `VAR_EXTERNAL` variables. We can rename the `is_var_external` to something else if you have a suggestion, but I think `is_external` should continue to refer to `LinkageType::External`.
@ghaith any thoughts?

Reply:
Yeah maybe you're right, especially since `VAR_EXTERNAL`s are not supported anyways